src/Behat/MinkExtension/Context/MinkContext.php

```diff
@@ -44,9 +50,13 @@ public function visit($page)
     }
 
     /**
-     * Reloads current page.
+     * Reloads current page
+     * Example: Given I reload the page
```

Comment:
this is not a Given step. Please don't advocate usage in a Given step. It is semantically wrong (same should be checked for all other steps)